src/SFA.DAS.ApprenticeCommitments.UnitTests/RenewingCommitmentStatementTests/ApprenticeshipConfirmation.cs

```diff
@@ -42,18 +40,18 @@ public void When_apprenticeship_section_confirmation_status_is_not_set_Then_appr
         [TestCase(false)]
         public void When_apprenticeship_section_confirmation_status_is_set_And_no_change_to_apprenticeship_has_occurred_Then_apprenticeship_section_does_not_change_status(bool confirmationStatus)
         {
-            _existingCommitmentStatement.SetProperty(p => p.ApprenticeshipDetailsCorrect, confirmationStatus);
+            _existingCommitmentStatement.Confirm(new Confirmations { ApprenticeshipDetailsCorrect = confirmationStatus }, DateTime.UtcNow);
```

Comment:
At one point in the development I had a whole extra class represent "Confirmed" and so SetProperty didn't work.

Turns out I didn't need that in the end, but setting the state via the proper API seems a good thing to do anyway.